src/Snppts/Snippets/ListOverviewTemplate.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Snppts.Authors;
+using Snppts.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Snppts.Snippets
+{
+    public class ListOverviewTemplate : IAmASnippet
+    {
+        public string Slug => "Listview-Item-Template";
+        public string Title => "Listview General Item Template";
+        public string GithubRepoName => "ListViewTemplate";
```

Comment:
`GithubRepoName` should contains with your github workspace name, like below:

```suggestion
        public string GithubRepoName => "attrib75/ListViewTemplate";
```
This is used to create a link to your source code, and currently not working:

<p align="center">
	<kbd>
		<img src="https://user-images.githubusercontent.com/19656249/68505965-431b0800-0247-11ea-8db9-6bb8c6bdd3c7.png" alt="image" style="max-width:100%;"/>
	</kbd>
</p>

Reply:
Ok, what should I do? change my code and submit another request?

Reply:
Here you can accept my suggestion, but yout can make this changes in your PC with your IDE and commit it at your forked repository branch (currently master)..
Are you using command line git? you can use [github desktop](https://desktop.github.com/), it suggest you to send more changes to a specific openned PR

More about contributing to projects:

[Get started](https://help.github.com/en/desktop/contributing-to-projects)
[Committing and reviewing changes](https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project)